release.sh

```diff
@@ -48,9 +48,9 @@ package() {
 	
 	if [[ $1 == "darwin" ]]; then
 		mv "$ui/Electron.app" "$ui/Sia.app"
-		rm -r "$ui/Sia.app/Contents/Resources/default_app"
-		cp -R ../app/ "$ui/Sia.app/Contents/Resources/app/"
-		mkdir -p "$ui/Sia.app/Contents/Resources/app/Sia"
+		rm -rf "$ui/Sia.app/Contents/Resources/default_app"
+		rsync -av --progress ../ "$ui/Sia.app/Contents/Resources/app/" --exclude "release/" --exclude ".git/"
+		mkdir -p "$ui/Sia.app/Contents/Resources/app/app/Sia"
```

Comment:
Noted, though, try this out and execute building a release and see if it poses any problems. For now, master's release.sh doesn't work so this is a short-term solution.